apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/TopBar.jsx

```diff
@@ -113,33 +115,73 @@ export type Props = {
   onReload: Function,
   onClose?: Function,
   onHelp?: Function,
-  onOpenDevTools: Function,
   config?: TopBarConfig,
+  webview: WebviewTag,
 };
 
 const WebPlatformTopBar = ({
   manifest,
   onReload,
   onHelp,
   onClose,
-  onOpenDevTools,
   config = {},
+  webview,
 }: Props) => {
   const { name, icon } = manifest;
 
   const {
     shouldDisplayName = true,
     shouldDisplayInfo = true,
     shouldDisplayClose = !!onClose,
+    shouldDisplayNavigation = false,
   } = config;
 
+  const [canGoBack, setCanGoBack] = React.useState(false);
+  const [canGoForward, setCanGoForward] = React.useState(false);
```

Comment:
it appears that `canGoBack  === canGoForward` is kept true at any point in time, so you only need one state?

Reply:
I need to keep track of both cases, to enable or disable the "forward" or "backward" button accordingly.

Furthermore, `canGoBack === canGoForward` is not true at all time. Sometimes you can go back but not forward, and some other times you can go forward but not back. And sometimes you can do both